README.md

```diff
@@ -1,109 +1,56 @@
-![Discord4J Logo](/images/d4j_l.png?raw=true)
+![Discord4J Logo](../images/d4j_l.png?raw=true)
 
-# Discord4J *v3* [![Download](https://jitpack.io/v/Discord4J/Discord4j.svg?style=flat-square)](https://jitpack.io/#Discord4J/Discord4j)  [![Support Server Invite](https://img.shields.io/badge/Join-Discord4J-7289DA.svg?style=flat-square)](https://discord.gg/NxGAeCY) [![Documentation Status](https://readthedocs.org/projects/discord4j/badge/?version=latest)](http://discord4j.readthedocs.io/en/latest/?badge=latest&style=flat-square)
+# Discord4J [![Maven Central](https://img.shields.io/maven-central/v/com.discord4j/Discord4J/2.svg?style=flat-square)](https://mvnrepository.com/artifact/com.discord4j/Discord4J)  [![JCenter](https://img.shields.io/bintray/v/austinv11/maven/Discord4J.svg?style=flat-square)](https://bintray.com/austinv11/maven/Discord4J/_latestVersion)  [![Support Server Invite](https://img.shields.io/badge/Join-Discord4J-7289DA.svg?style=flat-square&logo=discord)](https://discord.gg/NxGAeCY)
```

Comment:
Jitpack may still be a useful badge (not entirely sure)

Reply:
I think we are abandoning it before release, no?

Reply:
Not clear yet, @quanticc how do you evaluate artifactory for serving snapshots? I still think jitpack may be handy since its simpler to refer to commit tags than build numbers